### ISSUE TITLE
"Ansi" is incorrect capitalization in manual

### ISSUE THREAD

**Issue by tersec** [nim-lang/website#279]
https://nim-lang.org/docs/manual.html#types-cstring-type states:
> For the C backend the cstring type represents a pointer to a zero-terminated char array compatible with the type char* in Ansi C.

https://ansi.org/ is very clear that it's the "American National Standards Institute - ANSI".

**Comment by Araq** [comment]
Yeah, but most people don't understand how natural language works. I pronounce it as "ansi C", I don't say "A-N-S-I C" so the abbreviation became its own word and words are not written in ALLCAPS. Just saying, we can still change the website anyway.
